python-avd/pyavd/_eos_cli_config_gen/j2templates/eos-device-documentation.j2

```diff
@@ -121,6 +121,8 @@
 {% include 'documentation/quality-of-service.j2' %}
 {# InfluxDB Telemetry #}
 {% include 'documentation/monitor-telemetry-influx.j2' %}
+{# Monitor Server Radius #}
```

Comment:
There is a file called `monitoring.j2` which contains all the other similar monitor features. Please move it there and update the headings to being from ###.

Reply:
Done.